templates/core/login.mustache

```diff
@@ -205,7 +205,7 @@
                 </form>
 
                 <div class="forgetpass m-t-1">
-                    <p><a href="{{forgotpasswordurl}}">{{#str}}forgotten{{/str}}</a></p>
+                    <p><a href="{{forgotpasswordurl}}">Already have your account? <!-- {{#str}}forgotten{{/str}} --></a></p>
```

Comment:
This value should be derived from the 'forgotten' field in the language file. Otherwise this breaks multiple language support.

Reply:
```php
<p><a href="{{forgotpasswordurl}}">{{#str}} forgotten, theme_moove {{/str}}</a></p>
```

Reply:
https://docs.moodle.org/dev/Templates#.7B.7B.23_str_.7D.7D